locales/en.js

```diff
@@ -19,6 +19,11 @@ export default {
   joinRoomNamePlaceholder: 'Your name',
   joinRoomButton: 'Join Room',
   //
+  //Room
+  showButton: 'Show',
```

Comment:
Would it be possible to call them by what they do? so for eg. showCards or showVotes or showPoint etc.